### ISSUE TITLE
Question display of Operator on boundary=marker

### ISSUE THREAD

**Issue by Jasskurn** [SomeoneElseOSM/SomeoneElse-style#23]
When boundary=marker is shown, text from operator= is shown. I've seen this for boundary posts in Dartmoor marking the boundaries of the military training area. It arguably clutters the map.

[https://www.openstreetmap.org/node/1312872895](https://www.openstreetmap.org/node/1312872895
)
[https://map.atownsend.org.uk/maps/map/map.html#18/50.63167/-3.95238](https://map.atownsend.org.uk/maps/map/map.html#18/50.63167/-3.95238)

![marker n operator issue - map_andytownsend](https://github.com/user-attachments/assets/56fa724e-00d7-4440-9075-d0be51327c65)

In the UK, boundary=marker & operator=* is rare. Overpass shows it's mainly the markers on Dartmoor, and several in the East of England. 
[https://overpass-turbo.eu/s/1ReI](https://overpass-turbo.eu/s/1ReI)

Looking at other markers it appears the text shown is the operator, and the name or description

[https://www.openstreetmap.org/node/11960798203#map=19/52.378548/1.027859
](https://www.openstreetmap.org/node/11960798203#map=19/52.378548/1.027859
)
[https://map.atownsend.org.uk/maps/map/map.html#18/52.37855/1.02724](https://map.atownsend.org.uk/maps/map/map.html#18/52.37855/1.02724)

I feel this arguably clutters the map, and would suggest dropping the operator tag.
But I can see merit in the display of data, and it can be argued this issue is simply the large number of nearby markers. An issue that could happen with any object that occurs in large numbers in small areas.

**Comment by SomeoneElseOSM** [comment]
Yes, that doesn't look ideal.  What would make more sense would be to either (a) make it red instead of brown (as already happens for military signs and some other military stuff) and/or (b) exclude boundary markers from "operator" tagging (some items are already excluded).

I'll have a think about it.

**Comment by SomeoneElseOSM** [comment]
This was fixed in https://github.com/SomeoneElseOSM/SomeoneElse-style/commit/d51bad8a0cc2cf0ba792994d0c949d9feeee60f5 .  It'll appear on the raster map at https://map.atownsend.org.uk/maps/map/map.html#17/50.63186/-3.95453 the next time I do a build there (likely in a few days time), and on the vector map at https://map.atownsend.org.uk/vector/index.html#16/50.63186/-3.95453 (likely in a couple of days).
